bom/pom.xml

```diff
@@ -27,7 +27,7 @@
         <infinispanVersion>8.0.2.Final</infinispanVersion>
         <mongodbVersion>3.0.3</mongodbVersion>
         <fongodbVersion>2.0.2</fongodbVersion>
-        <orientdbVersion>2.1.9</orientdbVersion>
+        <orientdbVersion>[2.1.12,)</orientdbVersion>
```

Comment:
As a reminder: we don't use range version, use 2.1.12
This should be a separate comimt.